src/pal/src/pal_enclave.c

```diff
@@ -51,6 +65,60 @@ static const char *get_enclave_absolute_path(const char *instance_dir) {
     return (const char *)enclave_path;
 }
 
+
+static const unsigned char base64_table[65] =
+    "ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz0123456789+/";
+
+size_t base64_decode_len(const char *b64input) {
+    size_t len = strlen(b64input), padding = 0;
+
+    if (b64input[len - 1] == '=' && b64input[len - 2] == '=') { //last two chars are =
+        padding = 2;
+    } else if (b64input[len - 1] == '=') { //last char is =
+        padding = 1;
+    }
+
+    return (len * 3) / 4 - padding;
+}
+
+/**
+ * base64_decode - Base64 decode
+ */
+void base64_decode(const char *b64input, unsigned char *dest, size_t dest_len) {
```

Comment:
Please clarify the license of the base64 function.

Reply:
Will do